### ISSUE TITLE
MP4 is seen as WebM

### ISSUE THREAD

**Issue by francoism90** [kaltura/nginx-vod-module#1499]
I'm experimenting with AV1, as Handbrake supports is as release 1.7.0.

From what I've understood, webm is unsupported by this module, so I'm using mp4 as container.
However, when trying to play the file, the manifest is reporting this falsely as `video/webm` file, resulting in an unplayable video/audio.

Is this expected? Should I force a setting?

I'm using Dash, and I've tried AV1 10-bit (SVT) and Opus/AAC.

Thanks!

**Comment by erankor** [comment]
Yes, it is expected, the codecs - av1, vp9, vp8, vorbis, opus are returned in webm container in DASH, regardless of the source container. 
If it doesn't play, the problem is somewhere else, maybe the device doesn't support this profile/pixel depth/...

**Comment by francoism90** [comment]
@erankor Thanks for your reply. :)

Just to be sure:
- Does the file also needs the `.webm` extension?
- I've read in other tickets, the module only supports mp4-containers, is this correct? Because I'm forcing `mp4` as the container.

I'll check the file as well, because it should be playable directly in the browser.

**Comment by erankor** [comment]
The module has experimental support for webm/MKV - it works for some files, but it's definitely not complete. 
It is recommended to use MP4 container for all codecs.
